content/blog/2020-07-10-july-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,150 @@
+approach to using GitHub Actions & GitLab CI to start using continuous
+integration in ML model training- and it turns out, this conference is generally
+very excited about GitHub Actions! In another talk, GitHub technical advocate
+[Jonathan Peck](https://www.linkedin.com/in/peckjon/), shared some great use
+cases, including how he used an Nvidia Jetson Nano to run Actions. Generally,
+the mood around GitHub Actions in the ML community is very encouraging for a
+future with more automated data science practices.
+
+Extremely early the next morning, I co-led a workshop with clinician-scientist
+[Cris Lanting](https://www.linkedin.com/in/crislanting/?originalSubdomain=nl)
+about developing strong computational infrastructure and practices in research
+as part of the
+[Virtual Conference on Computational Audiology](https://computationalaudiology.com/).
+
+In addition to outlining key components of scientific reproducibility and
+project management, Cris and I did some investigating. We conducted an informal
+survey asking about the biggest barriers to making research reproducible and
+shareable. Participants overwhelmingly answered that they lack time and
+incentives. Difficulty managing dependencies (both data and software) was
+another popular answer. Sounds a bit like softare before
+[DevOps](https://en.wikipedia.org/wiki/DevOps)... For the curious, the workshop
+is still viewable!
+
+https://www.youtube.com/watch?v=W4CoptalWw0
+
+Lastly, check out our virtual poster for
+[SciPy 2020](https://www.scipy2020.scipy.org/)! We prepared a demo about
+[packaging models and datasets like software](https://dvc.org/blog/scipy-2020-dvc-poster)
+so they can be widely disseminated via GitHub.
+
+## Coming up soon
+
+Some things to watch out for in the next few weeks:
+
+- **DVC Virtual Meetup.** In May, we had our
+  [first every virtual meetup](http://localhost:8000/blog/may-20-dvc-heartbeat).
+  We had amazing talks from [Dean Pleban](https://twitter.com/DeanPlbn) and
+  [Elizabeth Hutton](https://github.com/ehutt), plus time for Q&A with the DVC
+  team- you can
+  [watch the recording](https://www.youtube.com/watch?v=19GMtrFykSU&list=PLVeJCYrrCemiOc1SS_PIB3Tb3HX0Aqw3j)
+  if you missed it!
+
+  On Thursday, July 30, we're hosting our second meetup! Ambassador
+  [Marcel Ribeiro-Dantas](http://mribeirodantas.me/) is hosting once again.
+  We'll have short talks about causal modeling and CI/CD, plus lots of time for
+  chatting and catching up. Please RSVP!
+
+<blockquote class="embedly-card"><h4><a href="https://www.meetup.com/DVC-Community-Virtual-Meetups/events/271844501/">July DVC Meetup: Data Science & DevOps!</a></h4><p>This meetup will be hosted by DVC Ambassador Marcel! AGENDA:We have two 10-minute talks on the agenda:- Causal Modeling with DVC - Marcel- Continuous integration for ML case studies - Elle Following talks, we'll have Q&A with the DVC team and time for community discussion.</p></blockquote>
```

Comment:
regular embed twitter/youtube-like didn't work?

Reply:
no it didn't work :(